extensions/eclipse-che-theia-about/package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "@eclipse-che/theia-about",
```

Comment:
It is up to you, but to me `@eclipse-che/che-theia-about` looks more consistent.

Reply:
@mmorhun I tried to be consistent with existing extensions
https://github.com/eclipse/che-theia/blob/master/extensions/eclipse-che-theia-plugin-ext/package.json#L2

https://github.com/eclipse/che-theia/blob/master/extensions/eclipse-che-theia-dashboard/package.json#L2

it seems they're using the pattern `@eclipse-che/theia-something`

Reply:
I see. My point was that `@eclipse-che` is namespace and then we need to have a component which is Che Theia About, like you have in class and file names: `AboutCheTheiaDialog`, `about-che-theia-dialog.ts`